ThunderCloud/EmbeddedLinksInputCheckItemCell.swift

```diff
@@ -37,4 +37,34 @@ open class EmbeddedLinksInputCheckItemCell: StormTableViewCell {
 	required public init?(coder aDecoder: NSCoder) {
 		super.init(coder: aDecoder)
 	}
+    
+    override open var accessibilityTraits: UIAccessibilityTraits {
+        get {
+            return checkView.isOn ? [.selected, .button] : [.button]
+        }
+        set { }
+    }
+    
+    override open var isAccessibilityElement: Bool {
+        get {
+            return true
+        }
+        set { }
+    }
+    
+    override open var accessibilityLabel: String? {
+        get {
+            return cellTextLabel?.text
+        }
+        set { }
+    }
+    
+    override open var accessibilityHint: String? {
+        get {
+            return checkView.isOn ?
+                "Selectable. Double tap to de-select".localised(with: "_CHECKITEM_VOICEOVER_HINT_SELECTED") :
```

Comment:
Slightly insane how we don't get things like this for free from iOS! Is this because the entire cell needs the hint, instead of just a component within it? I remember at iOSDevUK they did cover this, might have been from a SwiftUI perspective mind...

Reply:
Yeah it's a PITA! as far as I could see there was no `selectable` accessibility trait provided by the OS :(